### ISSUE TITLE
Handling a couple of errors

### ISSUE THREAD

**Issue by Bachelar** [workspacer/workspacer#281]
Hi! I'm trying to customize some things based on instructions provided, mainly minimizing windows and keybinds, but I get errors: Microsoft.CodeAnalysis.Scripting.CompilationErrorException: (312,13): error CS1061 when trying to set CanMinimizeWindows to „true” and Microsoft.CodeAnalysis.Scripting.CompilationErrorException: (318,37): error CS0104 „Element Keys is an ambiguous reference between „System.Windows.Forms.Keys” and workspacer.Keys” when trying to set my own keybinds.

Love to hear some advice on that, because I love workspacer.

**Comment by josteink** [comment]
Not sure about the first one, but it indicates some sort of error in your code (referring to non-existent properties on objects, most likely).

> „Element Keys is an ambiguous reference between „System.Windows.Forms.Keys” and workspacer.Keys” when trying to set my own keybinds.

This means that both the classes `System.Windows.Forms.Keys` and `workspacer.Keys` are in scope at the same time, so simply referring to "Keys" is not specific enough for the C# compiler to determine which one of those you are actually intending to use. Replacing `Keys` in your code with `workspacer.Keys` should solve that error.

**Comment by Bachelar** [comment]
Yep, problem with custom keybinds is no more, thank you from the bottom of my heart. Still trying my hand in that weird minimizing windows issue. I'm sure it's on my end, which is both paradoxically enlightening and frustrating. Thank you once again!

**Comment by josteink** [comment]
Without seeing your full config, it's hard to provide any more concrete advice really.
